### PR TITLE
solution

### PR DESCRIPTION
**Pull request by NikitaKarbazin**
[DEMO LINK](https://nikitakarbazin.github.io/react_people-table-basics/)

### CODE REVIEW

**Comment by NikitaKarbazin** [comment]
Забыл добавить контейнер! Думаю это не критично)

**Review by GavrilyukYura — APPROVED** [review]
Nice!

File: src/App.tsx [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
+        />
+        <Route
+          path="/"
+          element={<HomePage />}
+        />
+        <Route
+          path="*"
+          element={<NotFoundPage />}
+        />
+      </Routes>
```
Comment (illiarerun) [inline src/App.tsx:26]: Use [Navigate](https://reactrouter.com/docs/en/v6/components/navigate)) component to redirect from /home to /;

**Review by illiarerun — APPROVED** [review]
One comment left:

**Pull request closed without merging**